scripts/list-pause-plans.js

```diff
@@ -0,0 +1,146 @@
+import { ethers } from 'ethers';
+import { table } from 'table';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import pauseABI from './pause_abi.json' with { type: 'json' };
+
+const argv = yargs(hideBin(process.argv))
+    .option('all', {
+        alias: 'a',
+        type: 'boolean',
+        description: 'Show all plans',
+        default: false
+    })
+    .option('pending', {
+        alias: 'p',
+        type: 'boolean',
+        description: 'Show only pending plans',
+        default: false
+    })
+    .check(argv => {
+        if (argv.all && argv.pending) {
+            throw new Error("You cannot use --all and --pending together.");
+        }
+        return true;
+    })
+    .help()
+    .alias('help', 'h')
+    .argv;
+
+const PLOT_TOPIC = "0x46d2fbbb00000000000000000000000000000000000000000000000000000000";
+const EXEC_TOPIC = "0x168ccd6700000000000000000000000000000000000000000000000000000000";
+const DROP_TOPIC = "0x162c7de300000000000000000000000000000000000000000000000000000000";
+const MCD_PAUSE = "0xbE286431454714F511008713973d3B053A2d38f3";
+
+const tableConfig = {
+    columns: {
+        0: { width: 66, wrapWord: true },
+        1: { width: 42, wrapWord: true },
+        2: { width: 66, wrapWord: true },
+        3: { width: 33, wrapWord: true },
+        4: { width: 10, wrapWord: true },
+        5: { width: 10, wrapWord: true },
+        6: { width: 10, wrapWord: true }
```

Comment:
At least in my screen, which is WQHD (2560x1440), table rows cannot be displayed in one single line, making it harder to inspect the result.

I'd suggest reducing the size of some columns, even if that means cells must be split into multiple lines. 

```suggestion
        0: { width: 33, wrapWord: true },
        1: { width: 42, wrapWord: true },
        2: { width: 33, wrapWord: true },
        3: { width: 10, wrapWord: true },
        4: { width: 10, wrapWord: true },
        5: { width: 10, wrapWord: true },
        6: { width: 10, wrapWord: true }
```

Reply:
Done in [5db40df](https://github.com/makerdao/protego/pull/5/commits/5db40dfa4f84d0995476509911dc9856138743ee), The Spell column was added later on, So I cut down the address columns to 21 width (they are split in half like the others).